### ISSUE TITLE
Last Week in Pony - August 5, 2018

### ISSUE THREAD

**Issue by jdhorwitz** [ponylang/ponylang.github.io#321]

**Comment by dipinhora** [comment]
Pony 0.24.4 has been released. Upgrading is recommended: https://www.ponylang.org/blog/2018/07/0.24.4-released/

**Comment by patternspandemic** [comment]
Perhaps useful for _Last Week's Playground:_

**[Isolates and Tuples](https://playground.ponylang.org/?gist=16c4ae4851efea2d7903c4f5173fd12a)**
Working with isolates and tuples can be challenging given the extreme restriction of an `iso` capability. This playground covers putting isolates into a tuple, as well as extracting them back out.

---
As an aside: Perhaps it would be useful to keep a list of such playgrounds on the website, or create something similar for the tutorial. This way they won't get lost overtime in old Last Week in Pony posts, as they are not labelled in anyway as far as I can tell.

Actually, I can see _Last Week's Playground_ being transitioned to the tutorial, as a way to have such examples which don't exactly fit in with the flow of the tutorial. In the case of the above, I thought first to expand the tutorial, but including it in the tuples section felt too complicated (given refcaps are not yet introduced), and including it in any refcap section felt out of place.

Long story short:
Having a Playgrounds section in the tutorial with categories / labeled playgrounds, would make for better procurement and management of such playgrounds, and provide the tutorial with a way to expand on any feature or combination thereof without overloading the learner with inlined examples or out of context notes. Last Week in Pony could still feature interesting playgrounds procured there.

**Comment by SeanTAllen** [comment]
Do you use the Sublime Text Pony module? You might want to update it. Stefan Achatz has added a couple fixes in the last few weeks. Nothing huge, but... its an update. Didn't realize there was Sublime Text support? There is... feel free to contribute more fixes yourself at https://github.com/ponylang/sublime-pony.

**Comment by SeanTAllen** [comment]
The Ponylang emacs mode got an update recently thanks to [spreadLink](https://github.com/spreadlink). There's been no official release with the fix yet, but if you are running latest, you can take advantage of it now. 

A note from Sean, if you are interested in helping him as a maintainer of ponylang-mode, drop him an email [sean@monkeysnatchbanana.com](mailto:sean@monkeysnatchbanana.com).
